website/src/components/pageStateSelectors/wasap/filters/VariantExplorerFilter.tsx

```diff
@@ -70,6 +70,17 @@ export function VariantExplorerFilter({
                     step={0.01}
                     onChange={(v) => setPageState({ ...pageState, minJaccard: v })}
                 />
+                <LabeledField label='Time frame'>
+                    <select
+                        className='select select-bordered'
+                        value={pageState.timeFrame}
+                        onChange={(e) => setPageState({ ...pageState, timeFrame: e.target.value as VariantTimeFrame })}
+                    >
+                        <option value='all'>All</option>
+                        <option value='6months'>Past 6 months</option>
+                        <option value='3months'>Past 3 months</option>
```

Comment:
We essentially lose type safety here in the `as VariantTimeFrame` - let's maybe at least use constants so that the values of the options are coupled to the `VariantTimeFrame` type?
```suggestion
                    <select
                        className='select select-bordered'
                        value={pageState.timeFrame}
                        onChange={(e) => setPageState({ ...pageState, timeFrame: e.target.value as VariantTimeFrame })}
                    >
                        <option value={timeFrames.all}>All</option>
                        <option value={timeFrames.6months}>Past 6 months</option>
                        <option value={timeFrames.3months}>Past 3 months</option>
```

and then infer `VariantTimeFrame` from `timeFrames`.